druid/druid.sh

```diff
@@ -0,0 +1,511 @@
+#
+
+# This is not the full list of Druid extensions, but common ones that people often use. You may need to change this list
+# based on your particular setup.
+druid.extensions.loadList=["druid-kafka-eight", "druid-histogram", "druid-datasketches", "druid-lookups-cached-global", "mysql-metadata-storage", "druid-hdfs-storage","druid-kafka-indexing-service"]
+
+# If you have a different version of Hadoop, place your Hadoop client jar files in your hadoop-dependencies directory
+# and uncomment the line below to point to your directory.
+#druid.extensions.hadoopDependenciesDir=/my/dir/hadoop-dependencies
+
+#
+# Logging
+#
+
+# Log all runtime properties on startup. Disable to avoid logging properties on startup:
+druid.startup.logging.logProperties=true
+
+#
+# Zookeeper
+#
+
+druid.zk.service.host=${zookeeper_list}
+druid.zk.paths.base=/druid
+
+#
+# Metadata storage
+#
+
+
+# For MySQL:
+druid.metadata.storage.type=mysql
+druid.metadata.storage.connector.connectURI=jdbc:mysql://${sql_host}:3306/druid
+druid.metadata.storage.connector.user=druid
+druid.metadata.storage.connector.password=diurd
+druid.sql.enable = true
+
+# For PostgreSQL (make sure to additionally include the Postgres extension):
+#druid.metadata.storage.type=postgresql
+#druid.metadata.storage.connector.connectURI=jdbc:postgresql://db.example.com:5432/druid
+#druid.metadata.storage.connector.user=...
+#druid.metadata.storage.connector.password=...
+
+#
+# Deep storage
+#
+
+# For local disk (only viable in a cluster if this is a network mount):
+#druid.storage.type=local
+#druid.storage.storageDirectory=var/druid/segments
+
+# For HDFS (make sure to include the HDFS extension and that your Hadoop config files in the cp):
+druid.storage.type=hdfs
+druid.storage.storageDirectory=/druid/segments
+
+# For S3:
+#druid.storage.type=s3
+#druid.storage.bucket=your-bucket
+#druid.storage.baseKey=druid/segments
+#druid.s3.accessKey=...
+#druid.s3.secretKey=...
+
+#
+# Indexing service logs
+#
+
+# For local disk (only viable in a cluster if this is a network mount):
+#druid.indexer.logs.type=file
+#druid.indexer.logs.directory=var/druid/indexing-logs
+
+# For HDFS (make sure to include the HDFS extension and that your Hadoop config files in the cp):
+druid.indexer.logs.type=hdfs
+druid.indexer.logs.directory=/druid/indexing-logs
+
+# For S3:
+#druid.indexer.logs.type=s3
+#druid.indexer.logs.s3Bucket=your-bucket
+#druid.indexer.logs.s3Prefix=druid/indexing-logs
+
+#
+# Service discovery
+#
+
+druid.selectors.indexing.serviceName=druid/overlord
+druid.selectors.coordinator.serviceName=druid/coordinator
+
+#
+# Monitoring
+#
+
+druid.monitoring.monitors=["io.druid.java.util.metrics.JvmMonitor"]
+druid.emitter=logging
+druid.emitter.logging.logLevel=info
+
+# Storage type of double columns
+# ommiting this will lead to index double as float at the storage layer
+
+druid.indexing.doubleStorage=double
+EOF
+
+  cp /usr/lib/hadoop/etc/hadoop/core-site.xml conf/druid/_common/core-site.xml
```

Comment:
better to symlink these rather than copying.
s/cp/ln -s

Reply:
Already fixed:
`  ln -sf /usr/lib/hadoop/etc/hadoop/core-site.xml conf/druid/_common/core-site.xml
  ln -sf /usr/lib/hadoop/etc/hadoop/hdfs-site.xml conf/druid/_common/hdfs-site.xml
  ln -sf /usr/lib/hadoop/etc/hadoop/yarn-site.xml conf/druid/_common/yarn-site.xml
  ln -sf /usr/lib/hadoop/etc/hadoop/mapred-site.xml conf/druid/_common/mapred-site.xml`